### DISCUSSION TITLE
PHP本地运行环境

### DISCUSSION THREAD

**Discussion by flowerwOw0316** [1Panel-dev/1Panel#5741] [Q&A]
![image](https://github.com/1Panel-dev/1Panel/assets/50901029/f1e8c817-3d69-4d75-9f1c-b02272ddc7f3)
qing请问一下这个功能是怎么用的

**Comment by wanghe-fit2cloud** [comment]
参考：
https://www.bilibili.com/video/BV1Um4y1m7XQ/
